src/allmydata/crypto/rsa.py

```diff
@@ -0,0 +1,117 @@
+from cryptography.exceptions import InvalidSignature
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.asymmetric import rsa, padding
+from cryptography.hazmat.primitives.serialization import load_der_private_key, load_der_public_key, \
+    Encoding, PrivateFormat, PublicFormat, NoEncryption
+
+from allmydata.crypto import BadSignature
+
+
+class RsaMixin(object):
+
+    '''
+    This is the value that was used by `pycryptopp`, and we must continue to use it for
+    both backwards compatibility and interoperability.
+
+    The docs for `cryptography` suggest to use the constant defined at
+    `cryptography.hazmat.primitives.asymmetric.padding.PSS.MAX_LENGTH`, but this causes old
+    signatures to fail to validate.
+    '''
+    RSA_PSS_SALT_LENGTH = 32
+
```

Comment:
Could this just be a module-level constant instead and thus get rid of multiple-inheritance / "mixin" class?